qa/rpc-tests/test_framework/mc_test/mc_test.py

```diff
@@ -27,20 +27,42 @@ def __init__(self, datadir, srcdir, ps_type = "cob_marlin"):
         assert(ps_type == "darlin" or ps_type == "cob_marlin")
         self.ps_type = ps_type
 
-    def _generate_params(self, id, circuit_type, ps_type, file_prefix, num_constraints, segment_size):
-        params_dir = self._get_params_dir(id)
+    def _get_file_prefix(self, circ_type, constant):
+        if constant != None:
+            return str(self.ps_type) + "_" + circ_type + "_"
+        return str(self.ps_type) + "_" + circ_type + "_no_const_"
```

Comment:
This could be shortened to:
```suggestion
    def _get_file_prefix(self, circ_type, constant):
        return str(self.ps_type) + "_" + circ_type + ("_" if constant != None else "_no_const_")
```

Reply:
Applied your suggestion to 3518be16ce52befbdfc21398187b4961582107f2